mission_control/navigator_missions/nav_missions/circle.py

```diff
@@ -9,5 +8,5 @@ def main(navigator):
     while True:
         p += [0, 1, 0]
 
-        a = navigator.move.set_position(p).go(move_type='skid', initial_plan_time=0)
+        # a = navigator.move.set_position(p).go(move_type='skid', initial_plan_time=0)
```

Comment:
?

Reply:
F841: 'a' assigned but never used. I'll noqa this one